### DISCUSSION TITLE
HIIT Repetition

### DISCUSSION THREAD

**Discussion by bluebee1979** [wger-project/wger#723] [Ideas]
First of all, thanks for this wonderful software. Newbie here and tried to set-up work-out. Would like to ask a feature to group a set of exercises and the repetition is grouped as well. This is applicable for HIIT exercises.

**Comment by rolandgeider** [comment]
Hi! How do you mean grouped? It's possible make a super set by adding more than one exercise to a set

**Reply by bluebee1979** [reply]
Hi @rolandgeider thanks for the reply. When I add an exercise like below:

<html>
<body>


#1 | Run - Treadmill1 × 5 Kilometers
-- | --
#2 | Axe Hold10 (10 kg) – 10 (10 kg) – 10 (10 kg) – 10 (10 kg)
#3 | Crunches10 – 10 – 10 – 10
#4 | Dumbbell Concentration Curl10 (10 kg) – 10 (10 kg) – 10 (10 kg) – 10 (10 kg)
#5 | Hollow Hold10 – 10 – 10 – 10


</body>
</html>

When I execute this set, I will follow the repetition per exercise. Means after 1, I will do 4 reps of #2, followed by 4 reps of #3, etc...

Is there a way to put 1 rep of exercise 2-5, then another rep of exercise 2-5, so on?

**Reply by rolandgeider** [reply]
I see what you mean. That's exactly what the supersets will do, the gym mode will show you the first set of exercises 1, 2, 3 and 4, then the second set, the third

**Reply by rolandgeider** [reply]
With "set" I mean reps

**Reply by bluebee1979** [reply]
@rolandgeider thank you. I tried what you just described and it is working as per my expectation. Thank you very much for your help.

**Reply by bluebee1979** [reply]
For other reference, this is what I set-up



#1 | Run - Treadmill1 × 10
-- | --
#2 | Bentover Dumbbell Rows10 (15 kg) – 10 (15 kg) – 10 (15 kg) – 10 (15 kg)Crunches10 – 10 – 10 – 10Dumbbell Lunges Walking10 (15 kg) – 10 (15 kg) – 10 (15 kg) – 10 (15 kg)Push Ups10 – 10 – 10 – 10Keep your body straightGo as low as you can

**Reply by rolandgeider** [reply]
BTW, If you do for all reps the same, you can just fill in the first
